media-libs/openexr/openexr-2.5.2.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_ECLASS=cmake
+inherit cmake-multilib flag-o-matic toolchain-funcs
+
+DESCRIPTION="ILM's OpenEXR high dynamic-range image file format libraries"
+HOMEPAGE="https://www.openexr.com/"
+SRC_URI="https://github.com/AcademySoftwareFoundation/openexr/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0/25" # based on SONAME
+KEYWORDS="~amd64 -arm ~arm64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~amd64-linux ~x86-linux ~x64-macos ~x86-solaris"
+IUSE="cpu_flags_x86_avx doc examples static-libs utils test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	media-libs/ilmbase:=
+	sys-libs/zlib[${MULTILIB_USEDEP}]
+"
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig"
+
+S="${WORKDIR}/${P}/OpenEXR"
+
+DOCS=( PATENTS README.md )
+
+MULTILIB_WRAPPED_HEADERS=( /usr/include/OpenEXR/OpenEXRConfigInternal.h )
+
+src_prepare() {
+	cmake_src_prepare
+
+	# Fix path for testsuite
+	sed -i -e "s:/var/tmp/:${T}:" "${S}"/IlmImfTest/tmpDir.h || die "failed to set temp path for tests"
+
+	if use abi_x86_32 && use test; then
+		eapply "${FILESDIR}/${P}-0001-IlmImfTest-main.cpp-disable-tests.patch"
+	fi
+}
+
+multilib_src_configure() {
+	local mycmakeargs=(
+		-DBUILD_TESTING=$(usex test)
+		-DINSTALL_OPENEXR_DOCS=$(usex doc)
+		-DINSTALL_OPENEXR_EXAMPLES=$(usex examples)
+		-DOPENEXR_BUILD_BOTH_STATIC_SHARED=$(usex static-libs)
+		-DOPENEXR_BUILD_UTILS=$(usex utils)
+		-DOPENEXR_INSTALL_PKG_CONFIG=ON				# default
+	)
+
+	cmake_src_configure
+}
+
+multilib_src_install_all() {
+	if use doc; then
+		DOCS+=( doc/*.pdf )
+	fi
+	einstalldocs
+
+	if use examples; then
+		docompress -x /usr/share/doc/${PF}/examples
+	else
+		rm -r "${ED}"/usr/share/doc/${PF}/examples || die
```

Comment:
And indeed, now this happens:

`rm: cannot remove '/var/tmp/portage/media-libs/openexr-2.5.2/image/usr/share/doc/openexr-2.5.2/examples': No such file or directory`

I assume the directory does not exist because we already configured with `-DINSTALL_OPENEXR_EXAMPLES=OFF` in that case?

Reply:
Probably. Maybe I added the force flag because of such an error. IIRC it happened at some time that the directory was present in `${D}` although I hadn't passed the examples USE flag. I'm going to guard this line for now, instead of removing it clompletely.

Reply:
It just looks like an autotools remnant, line was there from 2.3.0.

Reply:
Yes, you're right. So you propose to remove the `else` clause completely?

Reply:
yup, you can even make a oneliner out of the docompress condition ;)